File: 02-backend-spring-boot-library/src/main/java/com/luv2code/spring_boot_library/requestmodels/AddHouseRequest.java
package com.luv2code.spring_boot_library.requestmodels;

import lombok.Data;

@Data
public class AddHouseRequest {

    private String title;

    private String LandLord;

    private String description;

    private int viewingSlots;

    private String category;

    private String img;

}

File: 03-frontend-react-app/src/models/AddHouseRequest.ts
class AddHouseRequest {
    title: string;
    landLord: string;
    description: string;
    viewingSlots: number;
    category: string;
    img?: string;

    constructor(title: string, landLord: string, description: string, viewingSlots: number, 
        category: string) {
            this.title = title;
            this.landLord = landLord;
            this.description = description;
            this.viewingSlots = viewingSlots;
            this.category = category;
        }
}

export default AddHouseRequest;